docs/resources/aws_ec2_launch_templates.md

```diff
@@ -0,0 +1,63 @@
+---
+title: About the aws_ec2_launch_templates Resource
+platform: aws
+---
+
+# aws\_ec2\_launch_\templates
+
+Use the `aws_ec2_launch_templates` InSpec audit resource to test properties of a all the AWS Launch templates.
+
+
+An `aws_ec2_launch_templates` resource block declares the tests for a all the  AWS Launch templates.
+
+describe aws_ec2_launch_templates do
+it { should exist }
+end
+
+#### Parameters
+
+One of either the launch_templates_name or launch_templates_id must be be provided.
```

Comment:
Is it launch_templates_name or launch_template_names?
Same question with launch_templates_id.